cypress/integration/createTrackedEntity.cy.js

```diff
@@ -75,37 +76,80 @@ const setUpTable = () => {
 
-    cy.getBySel('dimension-item-eventDate').should('not.exist')
-
+    // Check the correct time dimensions are displayed, and with program/stage specific name
     cy.getBySel(`dimension-item-${program.id}.enrollmentDate`).contains(
         program[DIMENSION_ID_ENROLLMENT_DATE]
     )
-
-    cy.getBySel('dimension-item-scheduledDate').should('not.exist')
-
     cy.getBySel(`dimension-item-${program.id}.incidentDate`).contains(
         program[DIMENSION_ID_INCIDENT_DATE]
     )
-
     cy.getBySel('dimension-item-lastUpdated').contains(
         program[DIMENSION_ID_LAST_UPDATED]
     )
+    cy.getBySel('dimension-item-eventDate').should('not.exist')
+    cy.getBySel('dimension-item-scheduledDate').should('not.exist')
 
+    // Add a program data dimension
     clickAddRemoveProgramDataDimension(programDataDimensionName)
 
-    // selectFixedPeriod({ label: periodLabel, period: TEST_FIX_PE_DEC_LAST_YEAR }) // FIXME: Time dimensions aren't implemented yet
+    // Adding time dimensions
+    const firstQuarterThisYear = {
+        type: 'Quarterly',
+        year: getCurrentYearStr(),
+        name: `January - March ${getCurrentYearStr()}`,
+    }
+    const secondQuarterThisYear = {
+        type: 'Quarterly',
+        year: getCurrentYearStr(),
+        name: `April - June ${getCurrentYearStr()}`,
+    }
+
+    // Add the first time dimension
+    selectFixedPeriod({
+        label: periodLabel,
+        period: firstQuarterThisYear,
+    })
+
+    // Check the time dimension chip and tooltip content
+    cy.getBySelLike('layout-chip').contains(periodLabel).trigger('mouseover')
+    cy.getBySel('layout-chip-tooltip-content').contains(
+        firstQuarterThisYear.name
+    )
+    cy.getBySelLike('layout-chip').contains(periodLabel).trigger('mouseout')
+    cy.getBySel('layout-chip-tooltip-content').should('not.exist')
+
+    // Add another time dimension
+    selectFixedPeriod({
+        label: periodLabel,
+        period: secondQuarterThisYear,
+        selected: firstQuarterThisYear,
+    })
+
+    // Check the time dimension chip and tooltip content for the new content
+    cy.getBySelLike('layout-chip').contains(periodLabel).trigger('mouseover')
+    cy.getBySel('layout-chip-tooltip-content').contains(
+        `Program: ${program.programName}`
+    )
+    cy.getBySel('layout-chip-tooltip-content').contains(
+        firstQuarterThisYear.name
+    )
+    cy.getBySel('layout-chip-tooltip-content').contains(
+        secondQuarterThisYear.name
+    )
+    cy.getBySelLike('layout-chip').contains(periodLabel).trigger('mouseout')
+    cy.getBySel('layout-chip-tooltip-content').should('not.exist')
 
     // Go back to person dimensions to verify that they're still listed properly
     cy.getBySel('main-sidebar').contains('Person dimensions').click()
     cy.getBySel('tracked-entity-dimensions-list')
         .children()
         .should('have.length', 33)
 
-    clickMenubarUpdateButton()
+    // clickMenubarUpdateButton()
```

Comment:
This can be uncommented now that the backend is fixed? https://dhis2.atlassian.net/browse/DHIS2-16708